charts/internal/gvisor/templates/runtimeclass-gvisor.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: node.k8s.io/v1beta1
+kind: RuntimeClass
+metadata:
+  name: gvisor
```

Comment:
Multiple gvisor installations for different worker pools will all reconciles/overwrite this object, right?

Reply:
see above.